hyperspy/tests/io/test_bcf.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import numpy as np
+
+from nose.tools import assert_true
+from hyperspy.io import load
+
+test_files = ['P45_instructively_packed_16bit_compressed.bcf',
+              'P45_12bit_packed_8bit.bcf',
+              'P45_the_default_job.bcf']  # for not compressed testing
+
+
+def test_load():
+    # test bcf from hyperspy load function level
+    # some of functions can be not covered
+    # it cant use cython parsing implementation, as it is not compiled
+    for thingy in test_files:
+        my_path = os.path.dirname(__file__)
+        filename = os.path.join(my_path, 'bcf_data', thingy)
+        print('testing bcf loading from hyperspy level...')
+        s = load(filename, downsample=2, cutoff_at_kV=10)
+        s = load(filename)
```

Comment:
@to266 , do you mean I should save few pixels on my machine to numpy file, so I could test if it is the same?... hm that means it can be tested if everything works fine on big-endian machines. 

Yes it is loaded twice, because the options or lack of it causes code to go a bit different ways.
In this way test covers larger part of code. I probably should separate it to two test functions, especially if I am going to test some pixels.